core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -741,6 +743,7 @@ object KafkaConfig {
     "then controller will use \"broker1.example.com:9094\" with security protocol \"SSL\" to connect to the broker.\n" +
     "If not explicitly configured, the default value will be null and there will be no dedicated endpoints for controller connections."
 
+  val SocketTcpNoDelayDoc = "Enable the Nagle algorithm (TCP_NODELAY) on broker sockets."
```

Comment:
TCP_NODELAY disables the nagle algorithm, doesn't it? I think true is a bit misleading then.